lib/codec/rle.ts

```diff
@@ -3,9 +3,9 @@
 // https://github.com/apache/parquet-format/blob/master/Encodings.md
```

Comment:
Updates to this file are just setting the interface to only what the things need. Much easier for fixing other types later